cabal.project

```diff
@@ -33,10 +33,6 @@ packages:
 package *
   ghc-options: -j8
 
--- Warnings as errors for local packages
-program-options
-  ghc-options: -Werror
-
```

Comment:
Must: not remove this

If you want to turn off `-Werror`, use `cabal configure` (see contributing guidelines).